### PR TITLE
Add moderator note

### PR DESCRIPTION
**Pull request by Elblinator**

### CODE REVIEW

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts [hunk 1]
```diff
@@ -142,6 +176,8 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
 
     private _listOfSpeakers: ViewListOfSpeakers | null = null;
 
+    private _contentObject = this._listOfSpeakers?.content_object;
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts:179]: Add type.

**Review by bastianjoel — CHANGES_REQUESTED** [review]
Unfinished. I don't think we should merge this in its current state.

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.html [hunk 2]
```diff
@@ -391,6 +391,47 @@
     </div>
 </mat-card>
 
+<!-- Moderator Note-->
+<mat-card class="spacer-bottom-60" [ngClass]="isEditing ? 'os-form-card' : 'os-card'" *ngIf="canSeeModerationNote">
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.html:395]: I don't think it makes sense to display an empty moderation note if the user cannot edit it. 

```suggestion
<mat-card class="spacer-bottom-60" [ngClass]="isEditing ? 'os-form-card' : 'os-card'" *ngIf="canSeeModerationNote && (canManageModerationNote || (moderatorNotes | async))">
```

**Review by luisa-beerboom — APPROVED** [review]

**Comment by luisa-beerboom** [comment]
Code LGTM

**Pull request merged**